app/views/drivers/show.html.erb

```diff
@@ -0,0 +1,55 @@
+
+<p><%= link_to "Return to driver List", drivers_path, class: "btn btn-secondary"%></p> 
+<p> <strong>Total Earnings:</strong> $<%= @driver.total_earnings.to_f / 100 %></p>
+<p> <strong>Average rating:</strong> <%= @driver.average_rating%></p>
+
+<table class="table table-hover">
+   <thead class="thead-dark">
+    <tr>
+      <th>Id</th>
+      <th>Name</th>
+      <th>Vin</th>
+      <th>Availability</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr class="driver_table">
+      <td><%= @driver.id %></td>
+      <td><%= @driver.name %></td>
+      <td><%= @driver.vin %></td>
+      <td><%= @driver.available %></td>
+      <td> <%= link_to "Edit", edit_driver_path(@driver) %></td>
+      <td> <%= link_to "Delete", driver_path(@driver), method: :delete, data: {confirm: "Are you sure?"}%></td>
+    </tr>
+  </tbody>
+</table>
+
+<h3><%=@driver.name %>'s Trips</h3>
+
+   <table class="table table-hover">
+     <thead class="thead-dark">
+      <tr>
+        <th>Trip id</th>
+        <th>Driver's Name</th>
+        <th>Passenger's Name</th>
+        <th>Date</th>
+        <th>Trip cost</th>
+        <th>Trip rating</th>
+      </tr>
+    </thead>
+    <tbody>
+    <% @driver.trips.each do |trip| %>
+    
+      <tr>
+      <td><%= link_to trip.id, trip_path(trip.id) %></td>
+      <td><%= link_to trip.driver.name, driver_path(trip.driver) %></td>
+      <td><%= link_to trip.passenger.name, passenger_path(trip.passenger.id) %></td>
+        <td><%= trip.date %></td>
+        <td><%= trip.cost.to_f / 100 %></td>
+        <td><%= trip.rating %></td>
+        <td class="btn-table"> <%= link_to "Edit", edit_trip_path(trip.id) %></td>
+        <td class="btn-table"><%= link_to "Delete", trip_path(trip.id), method: :delete, data: {confirm: "Are you sure?"} %></td>
+      </tr>
+    </tbody>
+    <%end%>
+</table>
```

Comment:
Just noting that this would be a **great** partial view which is shared by the driver and passenger's shared view